components/app/interface/settings/client/index.tsx

```diff
@@ -196,6 +197,14 @@ const Config: SettingsConfiguration<{ server: Server }> = {
           {
             id: "logout",
             icon: <BiSolidExit size={20} color={theme!.colours.error} />,
+            href: "/login",
+            onClick: () => {
+              const acli = getController("state");
+              const cli = getController("client").getCurrentClient();
+              console.log(cli);
+              acli.auth.removeSession(cli?.user?.id as string);
+              window.location.href = "/login";
```

Comment:
Should useNavigate from routing package at top of component, and then use navigate function instead of telling the browser to redirect us.